### DISCUSSION TITLE
Show content of multiple workspaces at the same time

### DISCUSSION THREAD

**Discussion by upsala1** [hyprwm/Hyprland#1679] [Q&A]
I used AwesomeWM for the last couple of years and in awesome you can toggle the visibility of multiple workspaces (I think they are named tags there) at the same time which is a feature I really enjoyed. How much work would it be to implement this via plugin or script? Or is that even already possible and I just don't understand how to do this? To give an example of what I actually mean. Let's say I have 2 workspaces ws1 (kitty terminal) and ws2 (firefox) and I am currently on wm2, I would rightclick ws1 on the bar (or use a hotkey) to show both the kitty terminal and firefox at the same time. If I'd rightclick ws1 again I would only show firefox.

**Comment by vaxerski** [comment]
you could make a plugin / script. Tags are not supported in hl.

**Reply by upsala1** [reply]
My skills are a little rusty but I'm willing to give it a shot. Do you have any suggestions on how to do this? Just a rough pointing into the right direction would be nice.

**Reply by vaxerski** [reply]
assign tags to every window manually and manipulate them with `movetoworkspace`

**Reply by upsala1** [reply]
Thanks, I'll give it a try
